### PR TITLE
✨ Add userId to sitemap

### PR DESCRIPTION
**Pull request by williamchong**
Temp solution: add some creator owned by api wallet to sitemap
we should be using creator and collector api in the future to obtain wallets

### CODE REVIEW

File: src/config/sitemap.js [hunk 1]
```diff
@@ -10,21 +10,28 @@ const LIKECOIN_NFT_API_WALLET = IS_TESTNET
...
+  const classes = [].concat(
+    ...[newClassRes, topClassRes].map(r => (r.data || {}).classes || [])
+  );
+  const classIds = classes.map(c => c.id);
+  const uniqueClassIdRoutes = [...new Set(classIds)].map(
+    id => `/nft/class/${id}`
+  );
+  const users = ((creatorRes.data || {}).creators || []).map(c => c.account);
+  const userIdRoutes = users.map(id => `/${id}`);
+  return uniqueClassIdRoutes.concat(userIdRoutes);
```
Comment (nwingt) [inline src/config/sitemap.js:34]: I think it is better to rename the const for readibilty
```suggestion
  const routes = [...new Set(classIds)].map(
    id => `/nft/class/${id}`
  );
  const users = ((creatorRes.data || {}).creators || []).map(c => c.account);
  const portfolioPageRoutes = users.map(id => `/${id}`);
  return routes.concat(portfolioPageRoutes);
```

Comment (nwingt) [reply src/config/sitemap.js:34]: btw should we make sure `portfolioPageRoutes` is unique as well?

**Review by nwingt — COMMENTED** [review]
commit message should be `✨ Add portfolio pages to sitemap`?

File: src/config/sitemap.js (hunk 1 above)
Comment (williamchong) [reply src/config/sitemap.js:34]: `likechain/likenft/v1/creator` does not return repeated id, but we can write a dedup here for future use

**Review by nwingt — APPROVED** [review]

**Pull request merged**